client/css/AddToCart.scss

```diff
@@ -42,4 +47,28 @@ option {
 .cartItemsWrapper{
   height: 200px;
   overflow: auto;
-}
\ No newline at end of file
+}
+
+.addToCart {
```

Comment:
I want to learn more about SASS now. It looks like when you use this notation "addToCart__btn" the term prior to the underscore is like a pseudoelement and the term after the underscore is a child of that element?

Reply:
Not so much a pseudo-element, but a reference to the parent selector (in this case `.addToCart` which references the container. Now all the child classes can use that same **addToCart** prefix separated by __ and their functionality like `.addToCart__btn` which allows you to group your CSS together succinctly by using `&__btn`.